### ISSUE TITLE
Missing value plots from imputeTS package

### ISSUE THREAD

**Issue by rdstern** [IDEMSInternational/R-Instat#5944]
The latest version of this package has added ggplot graphs for the missing values.  We should either add them to the existing missing dialogue, or add a new dialogue.  I think we could add them as (an) additional button to the existing dialogue.

The first is  `imputeTS::ggplot_na_distribution(Tmax,x_axis=date) `  

Here is an example of the resulting graph:
![image](https://user-images.githubusercontent.com/11226469/90953839-e354bd00-e466-11ea-8523-01e5dd9ca168.png)

It produces a ggplot graph, so it should be possible to facet by station, though my R skills were not up to that.

The second one is gap size.  Not so useful and not so easy to facet, but might be useful totalled over a number of stations.

`last_graph <- imputeTS::ggplot_na_gapsize(Tmax,include=TRUE)`

![image](https://user-images.githubusercontent.com/11226469/90954316-2d3fa200-e46b-11ea-8ae8-41408d293fb5.png)

The next option is one we really need! 
`last_graph <- imputeTS::ggplot_na_imputations(x_with_na=Tmax,x_with_i=Int_Tmax1,x_axis=date) `

![image](https://user-images.githubusercontent.com/11226469/90954665-21091400-e46e-11ea-8acf-0dd8929ac845.png)

It also has a third option of truth, for when you make some values missing to test a method.

Also plotly works fine, so here is the same graph after zooming in on a section:
![image](https://user-images.githubusercontent.com/11226469/90954731-fcfa0280-e46e-11ea-9095-5bd6340ee51e.png)

**Comment by rdstern** [comment]
I hoped there might be some comments on the suggestions above.  Perhaps they are in Taiga?  Anyway let me continue with a more definite set of suggestions.
a) We confirm that ggplot_na_distribution can be used with the usual ggplot facets?  See above.
b) We change the name of the Infill Missing Values in the menu to just Missing Values.  In the menu it might move to below the Display Daily, which goes above the line, i.e. with Inventory.  Missing Values goes below and is alone, i.e. we have a line below it as well.  
c) We continue with a single dialogue, which therefore has 4 buttons.  The first is **_Display_**, which is the first graph above.
d) Then there is **_Fit Single_** followed by **_Fit Neighbouring_**. 
e) Finally there is **_Show_** which is ggplot_na_imputations graphs.  Again, I hope this can work with facets, though usually we would look in detail, so it might be mainly after we have filtered to a single station.  Design given in next message.

**Comment by rdstern** [comment]
The Graphical display options mentioned above can have a design that is consistent with the design of the Climatic > PICSA > Rainfall Graph dialogue.

a) Usual data selector
b) Single receiver with label Observed:
c) Single receiver with label Imputed:
d) Single receiver with label True Values (Optional):
e) Single receiver automatically filled with Date variable and label X-Variable:  (As in PICSA rainfall graph.)
f) Single receiver with label Facet By (Optional): and by default filled with the Station variable.  
g) On the lhs (as in the PICSA rainfall) there are two buttons with **_Missing Options_** and Plot options.  
h) Then there is the Save Graph control (as in PICSA) and the usual Comment/OK control.
i) On the Missing options sub-dialogue the first tab (as in PICSA) is Titles.  (There I am not sure if the Captions will work), but most of the rest of this tab should work.
j) The other tabs could be Points, Lines, Legend, Theme, for the options in the function.

**Comment by shadrackkibet** [comment]
I asked the author of the package if the package supports faceting see his response [here](https://github.com/SteffenMoritz/imputeTS/issues/46#issuecomment-687562068). Being able to facet is important to us therefore I think it is worth having a wrapper around these functions to allow facetting.

**Comment by rdstern** [comment]
@shadrackkibet I agree with the faceting, and capability of handling multiple stations.  I didn't fully follow the explanation in his message.  It gave the impression of being limited to 4 stations.

**Comment by shadrackkibet** [comment]
I give my suggestions as shown in the screenshots below. We can add more options into the sub dialog, but initially, we start with providing "important" ones in the main dialog.

![image](https://user-images.githubusercontent.com/26301973/92321471-f5238c00-f032-11ea-8610-29d7961caf65.png)
![image](https://user-images.githubusercontent.com/26301973/92321478-fe145d80-f032-11ea-8546-dcf6cf3dcdcb.png)
![image](https://user-images.githubusercontent.com/26301973/92321491-200de000-f033-11ea-8b55-c9858e8c261f.png)

**Comment by shadrackkibet** [comment]
Another plot provided in this package below. `ggplot_na_intervals()`

![image](https://user-images.githubusercontent.com/26301973/92321720-ee961400-f034-11ea-86ea-af16f2a57f9b.png)

**Comment by rdstern** [comment]
This looks really good.  Minor suggestions only.
a) Change the label "Observed:" to "Element:".  That's what we use on the climatic summary dialogue for example.
b) The Facet is presumably by station, so it is only enabled (or visible) when the Station field is filled.
c) It seems clear there will be a `Display Options` button later, so perhaps put it in its usual place now and disable.  There is space to move the Type and Save Graph controls down a bit on the existing dialogue.
d) (Being greedy now), but in the wrapper will it be possible to make the multiple graphs closer to facets, i.e. where the title and sub-title are only given once.  Being very greedy the x-axis is only at the bottom - though that may not be possible if the stations are of different lengths?

**Comment by lilyclements** [comment]
Fixed by #5938
